.github/workflows/ci.yml

```diff
@@ -0,0 +1,71 @@
+on: [push, pull_request]
+name: CI
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        name: [gcc, clang]
+        include:
+          - name: gcc
+            env:
+              CC: gcc
+              CXX: g++
+          - name: clang
+            env:
+              CC: clang
+              CXX: clang++
+
+    name: ${{ matrix.name }}
+    runs-on: [ubuntu-latest]
+
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - name: Environment
+      run: |
+        echo "::set-env name=CC::${{ matrix.env.CC }}"
+        echo "::set-env name=CXX::${{ matrix.env.CXX }}"
+
+    - name: Packages
+      run: |
+        sudo mkdir -p /usr/share/man/man1
```

Comment:
I'm curious why this is needed?

Reply:
Hmm, I was testing stuff with https://github.com/nektos/act and hit some issue related to installing some dependancy, but then I changed the "base" image I was working and didn't remove this.

It can (probably) go -- I'll check!